pkgs/top-level/release-haskell.nix

```diff
@@ -0,0 +1,206 @@
+/*
+  Test like this:
+  $ hydra-eval-jobs -I . pkgs/top-level/release-haskell.nix
```

Comment:
Would you be able to expand on this slightly?  For instance, what package is `hydra-eval-jobs` provided by?

On my NixOS system, `command-not-found` tells me that `hydra-eval-jobs` is provided by both the `hydra-migration` and `hydra-unstable` derivations.

Also, does this `hydra-eval-jobs` command kick off an actual job on Hydra?  Or does this just compile everything locally?

Reply:
Sure, I can another comment. You need `hydra-eval-jobs` from `hydra-unstable` and all the command does is print a jobset description in JSON format to `stdout`.